src/safir/metrics/_event_manager.py

```diff
@@ -49,23 +48,47 @@
 ]
 
 
+_UNRECOVERABLE_MSG = (
+    "The app metrics system is in an unrecoverable error state. No metrics"
+    " will be published by this app. When the underlying infrastructure"
+    " (Kafka, Schema Manager, etc) is healthy again, this app must be"
+    " restarted."
+)
+
+
 class _State(StrEnum):
```

Comment:
Not arguing exactly but if you're using auto() why bother to make it StrEnum instead of just Enum?